### ISSUE TITLE
The chatbot isn't working

### ISSUE THREAD

**Issue by vamsee9201** [metaskills/lambda-rag#1]
The interface is up and running. But, i don't think the API call is going to ChatGPT. There is no error message i can see on the terminal as well. It would be great if someone can help me with this.

**Comment by metaskills** [comment]
Did the Vue request make it to the Express backend? There is a request logging middle where we added in Express to answer just that question. https://github.com/metaskills/lambda-rag/blob/main/src/app.js#L15C9-L15C33

If not, then check your browser console for any errors If the request did make it to express, then maybe put some console.logs in place to verify your openai token is in place, if the route is working, etc. Does that help?

**Comment by vamsee9201** [comment]
No, the frontend isn't connecting to the backend. I couldn't see any log actually.
